alicloud/resource_alicloud_cs_kubernetes_permissions.go

```diff
@@ -0,0 +1,382 @@
+package alicloud
+
+import (
+	"strings"
+	"time"
+
+	"github.com/aliyun/terraform-provider-alicloud/alicloud/connectivity"
+
+	cs "github.com/alibabacloud-go/cs-20151215/v2/client"
+	"github.com/alibabacloud-go/tea/tea"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+)
+
+const ResourceName = "resource_alicloud_cs_kubernetes_permissions"
+
+func resourceAlicloudCSKubernetesPermissions() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAlicloudCSKubernetesPermissionsCreate,
+		Read:   resourceAlicloudCSKubernetesPermissionsRead,
+		Update: resourceAlicloudCSKubernetesPermissionsUpdate,
+		Delete: resourceAlicloudCSKubernetesPermissionsDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(90 * time.Minute),
+			Update: schema.DefaultTimeout(60 * time.Minute),
+			Delete: schema.DefaultTimeout(60 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"uid": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
the uid should be ForceNew

Reply:
https://github.com/aliyun/terraform-provider-alicloud/pull/3495/commits/a4f687532155d16a0c16e7a86406dd17ecc353aa